src/main/java/ch/uzh/ifi/hase/soprafs22/game/tiles/TileBuilder.java

```diff
@@ -71,13 +77,14 @@ public void setTileTraversingCost(int traversingCost) {
 
     //TODO create a UnitBuilder analogous to the TileBuilder
     @Override
-    public void setTileUnit(LinkedHashMap<String, Object> unitStream) {
+    public void setTileUnit(Map<String, Object> unitStream) {
         String unitType = (String) unitStream.get("type");
         Unit unit = new Unit();
         switch (unitType) {
             case "archer" -> unit.setType(UnitType.ARCHER);
```

Comment:
I don't know switches with -> but don't wee need "break;" else it triggers always default?

Reply:
nope, new fancy java syntax
(which I didn't know, IntelliJ let me shine^^)

Reply:
uhh nice, indeed very fancy 🎩 🧐